changelog/unreleased/bugfix-overlapping-view-requests

```diff
@@ -0,0 +1,10 @@
+Bugfix: Fix overlapping requests in files app
+
+In some cases the files app tended to display the wrong resources when navigation quickly through the views. This happened because the resource provisioning step wasn't canceld.
```

Comment:
```suggestion
In some cases the files app tended to display the wrong resources when navigating quickly through the views. This happened because the resource provisioning step wasn't canceled.
```